go.mod

```diff
@@ -183,6 +183,9 @@ require (
 	tags.cncf.io/container-device-interface v0.6.2 // indirect
 )
 
-// Fix an issue with fsutil v0.0.0-20230825212630-f09800878302 on Windows
-// See https://github.com/docker/buildx/issues/2207#issuecomment-1908460460
-replace github.com/tonistiigi/fsutil v0.0.0-20230825212630-f09800878302 => github.com/crazy-max/fsutil v0.0.0-20240124164449-376dc28ff40f
+replace (
+	// reverts https://github.com/moby/buildkit/pull/4094 to fix fsutil issues on Windows
+	github.com/docker/buildx => github.com/crazy-max/buildx v0.8.1-0.20240130141015-d7042ae5516c // compose-617f538cb315
+	github.com/moby/buildkit => github.com/crazy-max/buildkit v0.7.1-0.20240130133234-d9aa289bd124 // compose-957cb50df991
+	github.com/tonistiigi/fsutil => github.com/tonistiigi/fsutil v0.0.0-20230629203738-36ef4d8c0dbb
```

Comment:
If this is "ahead" of the one at https://github.com/docker/compose/blob/058200181006be6565108d8b9ab12ce9b0101d6f/go.mod#L140C2-L140C30


Then we probably don't need a replace rule (but wee could add a comment after the `// indirect` to indicate we manually bumped (temporarily)).

Reply:
Yes this was mainly to group them under the same context to keep track